### DISCUSSION TITLE
Make DevCard update every day in the same url

### DISCUSSION THREAD

**Discussion by jeancatarina** [dailydotdev/daily#1043] [Feature Requests]
Hello, I like DevCard but it is very annoying that we need to generate a new every day and change the URL in our website, would be nice if there is a fixed URL that automatically update every day

**Comment by idoshamun** [comment]
Hi @jeancatarina,

The DevCard is cached for 1 hour. So you can use the same URL but don't expect it to be updated faster than once per hour.
